src/EditorFeatures/CSharpTest/QuickInfo/SemanticQuickInfoSourceTests.cs

```diff
@@ -10663,4 +10663,118 @@ void M(IMyInterface i)
             """,
             MainDescription($"({CSharpFeaturesResources.awaitable}) ValueTask IAsyncDisposable.DisposeAsync()"));
     }
+
+    [Fact]
+    public async Task TestModernExtension1()
+    {
+        await TestWithOptionsAsync(
+            CSharpParseOptions.Default.WithLanguageVersion(LanguageVersion.Preview),
+            """
+            using System;
+            using System.Threading.Tasks;
+
+            static class Extensions
+            {
+                extension(string s)
+                {
+                    public void Goo() { }
+                }
+            }
+
+            class C
+            {
+                void M(string s)
+                {
+                    s.$$Goo();
+                }
+            }
+            """,
+            MainDescription($"void Extensions.extension(string).Goo() (+ 1 {FeaturesResources.overload})"));
```

Comment:
note: this is a bug.  we should not have 'overloads' here.  this is due to an odd semantic model result that i'm actively discussing with @jcouv .

Reply:
Can you at least add a comment to the effect this is considered a bug? Or file a tracking item?

Reply:
Please add a PROTOTYPE comment (those are allowed in feature branches and let us track some follow-up items)

Reply:
added